File: server/bs/prepare.go
package bs

type PrepareRequest struct {
	RoomID int `form:"room_id"`
}

type BasicLiveInfo struct {
	Title         string `json:"title"`
	LiveStartTime string `json:"live_start_time"`
	Cover         string `json:"cover"`
	LiveID        int64  `json:"live_id"`
}

type PrepareResponse struct {
	BasicLiveInfo
	LiveStatus int   `json:"live_status"`
	LightTs    int64 `json:"light_ts"`
}

File: www/src/mocks/prepare.js
const Prepare = {
    code: 0,
    msg: "",
    data: {
        live_status: 1,
        title: "直播标题",
        live_start_time: "2023-04-02 20:00:00 CST",
        cover: "/icon.png",
        live_id: 123456789,
        light_ts: 1234,
    }
}

export default {
    'get|^/api/prepare': opt => Prepare
}